Sources/HXPHPicker/Core/Util/PhotoManager+Download.swift

```diff
@@ -132,7 +132,8 @@ extension PhotoManager: URLSessionDownloadDelegate {
             }
         }else {
             let url: URL
-            if key.hasSuffix("mp4") || key.hasSuffix("MP4") {
+            // key.contains("mp4") || key.hasSuffix("MP4"): Consideration of videos with query parameter
+            if key.hasSuffix("mp4") || key.hasSuffix("MP4") || key.contains("mp4") || key.hasSuffix("MP4") {
```

Comment:
@TomoyaItagawa can you give the url string sample to explain why we need to add `key.contains("mp4")`.

Reply:
Also isn't `key.hasSuffix("MP4")` duplicated on this if clause?

Reply:
Exactly.
I thought it would be better to keep the prototype, but I'm going to change it to contains only.

fixed: [12376b3](https://github.com/jenald/HXPHPicker/pull/1/commits/12376b3cfa22b27a4339f5d36b21ed3eb92789bc)

Reply:
but if the url contains `mp4` in any part of the path then it will treated as a valid videoURL right? Is that okay?

Reply:
```suggestion
if responseURL.path.hasSuffix("mp4") || responseURL.path.hasSuffix("MP4") {
```